config.go

```diff
@@ -28,6 +28,9 @@ type config struct {
 	// calculated time will trigger a retry to ensure in use resources are not removed.
 	RetryOffset time.Duration `env:"RYUK_RETRY_OFFSET" envDefault:"-1s"`
 
+	// ChangesRetryInterval is the interval between retries if resource changes are detected.
+	ChangesRetryInterval time.Duration `env:"RYUK_CHANGES_RETRY_INTERVAL" envDefault:"1s"`
```

Comment:
We need to add this new setting to the readme. We must produce a minor for the new release then

Reply:
Question: do we want to be configurable, or just opinionated and enforce a fixed value?

Reply:
I considered that, I went with just having everything in the config for consistency, happy to revisit in a future PR.